config/Makefile.msvc

```diff
@@ -94,6 +94,12 @@ BYTECC=cl -nologo -D_CRT_SECURE_NO_DEPRECATE
 ### Additional compile-time options for $(BYTECC).  (For static linking.)
 BYTECCCOMPOPTS=-O2 -Gy- -MD
 
+### Additional compile-time options for $(BYTECC).  (For debug version.)
+BYTECCDBGCOMPOPTS=-Zi
+
+### Flag to use to rename object files.  (for debug version.)
+NAME_OBJ_FLAG=/Fo
```

Comment:
Actually I vaguely remember some recent Microsoft C compilers warning about the use of "-o" instead of "/Fo"...  Or was it for "/Fe" ? (executable name?).